UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -906,9 +906,9 @@ <h2>Discovering accessible content</h2>
 			<p>Publication providers, such as vendors and libraries create searching and filtering tools that interpret
 				accessibility metadata to aid in discovery. The set and variety of filters depend on the public they
 				address and the type of book they propose. </p>
-			<p> User feedback stated that in the missing of specific accessibility filters, product details like file
-				format and measures of protections are crucial information.</p>
-			<p>Taking into consideration those realities, the following sections are proposing a minimum and an
+			<p> User feedback stated    that missing  specific accessibility filters     focused on product details like file
+				format and measures of protection are crucial information.</p>
```

Comment:
I don't think you've captured the source sentence here. I'm kind of struggling with it myself, but I'd propose the following instead:
"User feedback indicates that in the absence of specific accessibility filters, they find product details like file format and protection measures (e.g., digital rights management) crucial information in determining usability."

```suggestion
			<p> User feedback indicates that in the absence of specific accessibility filters, they find product details like file
				format and protection measures (e.g., digital rights management) crucial information in determining usability.</p>
```